platform/api.visual/src/org/netbeans/api/visual/widget/SceneComponent.java

```diff
@@ -101,12 +101,7 @@ public void paint (Graphics g) {
 //        System.out.println ("CLIP: " + g.getClipBounds ());
 //        long s = System.currentTimeMillis ();
         Graphics2D gr = (Graphics2D) g;
-
-        Object props = Toolkit.getDefaultToolkit ().getDesktopProperty ("awt.font.desktophints"); // NOI18N
-        if (props instanceof Map)
-            gr.addRenderingHints ((Map) props);
-        gr.setRenderingHint (RenderingHints.KEY_ANTIALIASING, RenderingHints.VALUE_ANTIALIAS_ON);
-        gr.setRenderingHint (RenderingHints.KEY_TEXT_ANTIALIASING, RenderingHints.VALUE_TEXT_ANTIALIAS_ON);
```

Comment:
Here's another case with a slight change in behavior from before. Previously, standard text antialiasing would be applied, but now, subpixel antaliasing might be applied on Windows if it's enabled in the rest of the IDE. I think that's good.